src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -526,17 +527,23 @@ export default class RightPanel extends React.PureComponent<Props, State> {
     }
 
     const canEditItemMetadata = this.canEditItemMetadata(item)
+    const handleOpenVideoDialog = canEditItemMetadata
+      ? () => this.handleOpenVideoDialog()
+      : () => this.handleOpenVideoDialog({ viewOnly: true })
 
     return (
       <div className="details">
         {canEditItemMetadata ? (
           <>
-            <Icon name="edit" className="edit-item-file" onClick={this.handleOpenVideoDialog} />
-            <Icon name="play" className="play-video-button" onClick={this.handleOpenVideoDialog} />
-            <ItemVideo item={item} src={this.state.video} showMetrics onClick={this.handleOpenVideoDialog} />
+            <Icon name="edit" className="edit-item-file" onClick={handleOpenVideoDialog} />
+            <Icon name="play" className="play-video-button" onClick={handleOpenVideoDialog} />
+            <ItemVideo item={item} src={this.state.video} showMetrics onClick={handleOpenVideoDialog} />
           </>
         ) : (
-          <ItemVideo item={item} src={this.state.video} showMetrics />
+          <>
+            <Icon name="play" className="play-video-button" onClick={handleOpenVideoDialog} />
```

Comment:
could you please show this button only if the item has a video uploaded?

```suggestion
            {VIDEO_PATH in item.contents && (<Icon name="play" className="play-video-button" onClick={handleOpenVideoDialog} /> )}
```